engine/src/main/scala/cromwell/jobstore/SqlJobStore.scala

```diff
@@ -56,6 +56,7 @@ class SqlJobStore(sqlDatabase: JobStoreSqlDatabase) extends JobStore {
       case "Int" => WdlIntegerType
       case "Float" => WdlFloatType
       case "Boolean" => WdlBooleanType
+      case "File" => WdlFileType
```

Comment:
Still ToL: If we're starting to discuss "current code" vs. this "migration code", _in theory_ I may want a separation. The migration code shouldn't change. It will always migrate 0.19 to 0.21. Meanwhile current code will continue to change to suit the needs of the application.

That said this may be a non-issue for active users. And if there was someone migrating from <0.19 to a much later version, we did discuss that one could just be required to say run the binary cromwell 0.21-0.x _before_ running say cromwell 1.x.